src/NotionProvider-doc.ts

```diff
@@ -1,9 +1,28 @@
 /* Copyright © 2022 Seneca Project Contributors, MIT License. */
 
+const Joi = require('@hapi/joi')
+
 const docs = {
   get_info: {
     desc: 'Get information about the Notion SDK.',
   },
+
+  load_page: {
+    desc: 'Load Notion page data into an entity.',
+    examples: {
+    },
+    reply_desc: {
+    }
+  },
+
+  save_page: {
+    desc: 'Update/Save Notion page data into an entity.',
```

Comment:
This seems to be generating `Load Notion page data into an entity` into the README.md even though I specified what needs to be description there. An issue?

Reply:
hmm - check @seneca/provider - perhaps it is a bug where the wrong function name is being assigned

Reply:
will investigate.